.github/workflows/ci.yaml

```diff
@@ -218,6 +218,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          git config --global user.email "team@greatexpectations.io"
+          git config --global user.name "Great Expectations"
```

Comment:
This is the same user email/name that we set in Core for ci.yml before building the distribution and publishing to PyPI